### ISSUE TITLE
Does not build anymore

### ISSUE THREAD

**Issue by stefonarch** [lxqt/lxqt-connman-applet#17]
Build fails with a lots of "error: template with C linkage"
```
In file included from /usr/include/qt/QtCore/qmap.h:43:0,
                 from /usr/include/qt/QtCore/QMap:1,
                 from /home/stef/git/lxqt-connman-applet/strings.h:29,
                 from /usr/include/string.h:431,
                 from /usr/include/qt/QtCore/qarraydata.h:44,
                 from /usr/include/qt/QtCore/qbytearray.h:46,
                 from /usr/include/qt/QtCore/qmetatype.h:46,
                 from /usr/include/qt/QtCore/QMetaType:1,
                 from /home/stef/git/lxqt-connman-applet/connmanobject.h:28,
                 from /home/stef/git/lxqt-connman-applet/connmanobject.cpp:1:
/usr/include/qt/QtCore/qmap.h:1282:1: error: template with C linkage
 Q_DECLARE_ASSOCIATIVE_ITERATOR(Map)
 ^
/usr/include/qt/QtCore/qmap.h:1283:1: error: template with C linkage
 Q_DECLARE_MUTABLE_ASSOCIATIVE_ITERATOR(Map)
 ^
make[2]: *** [CMakeFiles/lxqt-connman-applet.dir/build.make:83: CMakeFiles/lxqt-connman-applet.dir/agent.cpp.o] Error 1
make[2]: *** [CMakeFiles/lxqt-connman-applet.dir/build.make:131: CMakeFiles/lxqt-connman-applet.dir/agentdialog.cpp.o] Error 1
make[2]: *** [CMakeFiles/lxqt-connman-applet.dir/build.make:155: CMakeFiles/lxqt-connman-applet.dir/connmanobject.cpp.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:68: CMakeFiles/lxqt-connman-applet.dir/all] Error 2
make: *** [Makefile:130: all] Error 2

```

##### System Information

* Distribution & Version: Arch & latest git checkout
* Kernel: 
* Qt Version: 5.9.1-3

**Comment by ghost** [comment]
I have the same issue, I am also using Arch

**Comment by yan12125** [comment]
Renaming lxqt-connman-applet/strings.h to something else should fix the problem. Apparently this file conflicts with /usr/include/strings.h. I don't use lxqt-connman-applet so I'm not sure whether this will break things otherwise or not.

**Comment by ghost** [comment]
@yan12125 You're right, this solve the issue

**Comment by stefonarch** [comment]
This does not work on my system...
 I'm using [CMST](https://github.com/andrew-bibb/cmst) which is much more complete.

**Comment by ghost** [comment]
@stefonarch well you also need to rename file in import statement

**Comment by stefonarch** [comment]
Ok, I missed some, it's also  in itemcontroller.cpp and agentdialog.cpp
@yan12125 could you make a commit for this?

**Comment by yan12125** [comment]
Fixed with #18. Thanks you @gabretana!
